### DISCUSSION TITLE
does this affect any settings with sound???

### DISCUSSION THREAD

**Discussion by Th1rtyS1x** [Raphire/Win11Debloat#107] [Q&A]
after a few days of having installed this my microphone has been acting really strange, it wont detect any sound unless i disable and re enable it but in doing so it causes windows to stop outputting audio at all and will only get fixed once i disable the microphone again, ive tested the mic on another pc and it works upon connecting it and even went and got a different mic and attached it to the same cable and that worked 

this only started to happen within the week of getting this and just now hit me to even consider asking about this

**Comment by Raphire** [comment]
Heya,

There have been 2 user reports of audio issues (#78) in the past. In that case I attributed it to dolby apps being removed by the script as some of these dolby apps seem to integrate with the audio drivers. For this reason the script no longer removes any dolby apps as of July 2. 

Did you run the script before or after this date? And additionally, what is the model of your PC/laptop?

**Reply by Th1rtyS1x** [reply]
most likely after as i remember hearing about the script rather recently within a week or 2

Device name	Kage
Processor	AMD Ryzen 5 5600X 6-Core Processor                3.70 GHz
Installed RAM	32.0 GB
Device ID	1A7A6443-7C9D-4F7F-BF99-CA7CE08EF55A
Product ID	00326-10003-47544-AA143
System type	64-bit operating system, x64-based processor
Pen and touch	No pen or touch input is available for this display

**Reply by Raphire** [reply]
In that case I am unsure what could have caused your issue. The script doesn't affect anything related to the sound settings or sound devices. 

Have you tried reinstalling the audio drivers? If it's a USB microphone you'll have to get them from their website. If it's a 3.5mm microphone that you plug into the back of your motherboard then you'll find the audio drivers on your motherboard manufacturer website.

**Comment by Th1rtyS1x** [comment]
ive tried reinstalling realtek drivers that hasnt fixed it
and the products drivers.... it doesnt have them
https://www.sudotack.com/Home/Products/detail/code/022601#details
you can look all you like but i cant find them at all nor did it require them when i got the mic

**Comment by Gawidev** [comment]
Speaking of audio issues, Ive run into an odd issue where the volume mixer no longer retains the state set to apps when theyre closed.
I've heard that in the past (win7) the values for the mixer were stored at
`HKEY_CURRENT_USER\Software\Microsoft\Internet Explorer\LowRegistry\Audio\PolicyConfig\PropertyStore`
but I'm not sure if this is still the case in windows 11.
It would make sense though since the "LowRegistry" entry does not exist on my machine, some part of me wonders if it is only created the first time an adjustment to per-application volume is done (which I hadn't done before removing IE through the debloat script).
@Raphire Have there been any other reports of audio issues since?

**Reply by Raphire** [reply]
Other than the issues I linked to in an earlier post I know of no other audio issues. I have encountered this myself in the past though, it seems to be app specific, sometimes it does remember the audio level, sometimes it doesn't.

It seems to be a Windows 11 issue however, as it's been a widely reported issue, just see these [results here](https://kagi.com/search?q=volume+mixer+not+saving+state+w11+site%3Areddit.com&r=nl&sh=L32nwrLxfaWg1TgrkYUU8Q).

**Reply by Gawidev** [reply]
Ah I see thanks, I've just reinstalled without the issue this time. Also that link is a search redirect which only works when logged in but im gratefuly it's made me discover a rather neat site 😊
Hope you have a nice day.

**Reply by Raphire** [reply]
Glad you managed to resolve it. Windows issues can be so hard to pin down. 

> Also that link is a search redirect which only works when logged in but im gratefuly it's made me discover a rather neat site 😊

Oof, I thought they allowed sharing now to actually show results. My bad!

> Hope you have a nice day.

You too!
